pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -660,8 +664,51 @@ func (m *ManagerImpl) allocateContainerResources(pod *v1.Pod, container *v1.Cont
 // for the passed-in <pod, container> and returns its DeviceRunContainerOptions
 // for the found one. An empty struct is returned in case no cached state is found.
 func (m *ManagerImpl) GetDeviceRunContainerOptions(pod *v1.Pod, container *v1.Container) *DeviceRunContainerOptions {
-	m.mutex.Lock()
-	defer m.mutex.Unlock()
+	podUID := string(pod.UID)
+	contName := container.Name
+	for k, v := range container.Resources.Limits {
+		resource := string(k)
+		needed := int(v.Value())
+		glog.V(3).Infof("needs %d %s", needed, resource)
```

Comment:
`Container XXX needs Y resources`

Reply:
done.

Reply:
Looks like you fixed the others but not this one